pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_torch_creation.cc

```diff
@@ -36,7 +36,7 @@ JNIEXPORT jlong JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_torchFromBlob(JNI
   }
   // Don't change device unless data on CPU
   if (!device.is_cpu()) {
-    result = result.to(device);
+    result = result.to(device, false, true);
```

Comment:
What is change for?

Reply:
we don't need this after I checked the source code